loguru.cpp

```diff
@@ -470,7 +470,13 @@ namespace loguru
 		for (int arg_it = 1; arg_it < argc; ++arg_it) {
 			auto cmd = argv[arg_it];
 			auto arg_len = strlen(verbosity_flag);
-			if (strncmp(cmd, verbosity_flag, arg_len) == 0 && !std::isalpha(cmd[arg_len], std::locale(""))) {
+			bool is_alpha;
+			try {
+				is_alpha = std::isalpha(cmd[arg_len], std::locale(""));
```

Comment:
Why are we calling locale with `""` and not using the default constructor? Would it make sense to just call `std::isalpha`?